### DISCUSSION TITLE
Unknown Error entering IP /Access Code

### DISCUSSION THREAD

**Discussion by donutslave** [greghesp/ha-bambulab#158] [General]
Any hints for getting past this error? Thanks! 
<img width="584" alt="Screenshot 2023-07-07 at 10 05 19 AM" src="https://github.com/greghesp/ha-bambulab/assets/138824124/c555cc39-83f0-40ac-a2f5-4634f5dae4c8">

**Comment by fonzy69** [comment]
Same problem for me, i have the ip and the access code but i'm not in Lan only mode, is it the problem?
Thanks for your help

**Comment by fonzy69** [comment]
tried in lan only mode and it failed to connect too

**Comment by greghesp** [comment]
Could you let us know if this is stil an issue please?

**Reply by Monsterfrog** [reply]
Hi. Im running into this issue on a X1 Carbon.

**Reply by cradermacher** [reply]
Same error for me. I can´t access my new X1C. I have testet it with both Versions, IP Adress an Bambu Account. Both don´t work.

**Reply by Monsterfrog** [reply]
Local wouldnt work for me so this error happened when i tried cloud connection. But then i tried local connection like 4 more times and on the 5th, it worked. Little strange.

**Reply by cradermacher** [reply]
ok, i will test it, thanks

**Reply by cradermacher** [reply]
Sorry, i have not an "unknown error".
My error is "failed to connect" on both ways local an cloud

**Comment by greghesp** [comment]
Could you follow up in https://github.com/greghesp/ha-bambulab/issues/245 please.

Will need more information such as debug logs and diagnostics before we can identify whats causing this
